src/Appwrite/Platform/Workers/Builds.php

```diff
@@ -233,6 +232,8 @@ protected function buildDeployment(Device $deviceForFunctions, Func $queueForFun
                         throw new \Exception('Unable to clone code repository: ' . $stderr);
                     }
 
+                    Console::execute('find ' . \escapeshellarg($tmpTemplateDirectory) . ' -type d -name ".git" -exec rm -rf {} +', '', $stdout, $stderr);
```

Comment:
Few reasons:
1. path traversal protection, it's less dangerous than using `rm -rf` directly as we're only operating within the`$tmpTemplateDirectory)` tree
2. this will also delete nested .git folders, such as in the case of submodules
3. it doesn't return exit code if the `.git` doesn't exist